src/test/java/pacman/WallandDotsTest.java

```diff
@@ -0,0 +1,13 @@
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class WallAndDotsTest {
+	@Test
+	public void testExist(){
+        int x=1;
+        int y=1;
+        wall wall1_1 = new wall(x,y, true);
```

Comment:
Hi, this is not compiling
![image](https://user-images.githubusercontent.com/4606607/87093278-f25f1300-c20a-11ea-9ee8-6223e3331b0c.png)